addons/rating/controllers/main.py

```diff
@@ -25,6 +25,14 @@ def action_open_rating(self, token, rate, **kwargs):
             raise ValueError(_("Incorrect rating: should be 1, 3 or 5 (received %d)"), rate)
 
         rating, record_sudo = self._get_rating_and_record(token)
+        session_partner = request.env.user.partner_id
+        if rating.partner_id != session_partner and \
+                session_partner in record_sudo.message_follower_ids.mapped('partner_id'):
+            values = {
```

Comment:
No need to defined a variable, you can inline the rendering params in the request.render call.